File: 4344.c
#include <stdio.h>

int main(){
	int c, n;
	int sum[10] = { 0 };
	int cnt = 0;
	int score[100];
	int av[10];
	float per[100];

	scanf("%d", &c); // 테스트 케이스의 개수

	for (int i = 0; i < c; i++){ //테스트 케이스의 개수만큼 학생수를 받음
		scanf("%d", &n); // 학생의 수
		for (int j = 0; j < n; j++){ // 학생의 수 만큼 점수를 받음
			scanf("%d", score[j]); //점수
			sum[i] += score[j]; // 0번째 케이스에서 학생들의 점수 총 합
			// 각 점수는 살려야하기 때문에 배열로 저장
			// 각 케이스의 합도 따로 저장해야 함
		}
		av[i] = sum[i] / n;

		if (av[i] < score[i]){
			cnt++;
		}

		per[i] = cnt / n * 100;
		printf("%6.3f", per[i]);
	}
	
	return 0;
}

File: 4344.py
'''
대학생 새내기들의 90%는 자신이 반에서 평균은 넘는다고 생각한다. 당신은 그들에게 슬픈 진실을 알려줘야 한다.
'''

C=int(input())
sum = 0
cnt = 0
allScore=[]
result=0

for i in range(C):
    score=list(map(int,input().split()))
    allScore.append(score)

for i in range(C):

    for j in range(1,len(allScore[i])):
        sum+=allScore[i][j]
    av=sum/allScore[i][0]


    for k in range(1,len(allScore[i])):
        if allScore[i][k]>av:
            cnt+=1

    result=cnt/(len(allScore[i])-1)*100
    print("%0.3f%%" %result)

    sum=0
    av=0
    cnt=0